packages/hardhat/test/integration-tests/periphery/controller-helper.ts

```diff
@@ -7,6 +7,7 @@ import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/dist/src/signer-wit
 import { WETH9, MockErc20, ShortPowerPerp, Controller, Oracle, WPowerPerp, ControllerHelper, INonfungiblePositionManager} from "../../../typechain";
 import { deployUniswapV3, deploySqueethCoreContracts, deployWETHAndDai, addWethDaiLiquidity, addSqueethLiquidity } from '../../setup'
 import { one, oracleScaleFactor, getNow } from "../../utils"
+import { convertCompilerOptionsFromJson } from "typescript";
```

Comment:
What does this do ?

Reply:
I believe @alpinechicken added this, I can remove, maybe related to above uniswap thing?

Reply:
I think it is not needed